Sources/ImgixSwift/include/CommonCrypto/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CommonCrypto {
+  header "/usr/include/CommonCrypto/CommonCrypto.h"
```

Comment:
I'm skeptical of hard coding a path here but this was the only I was able to get it work.